docs/sources/features/explore/index.md

```diff
@@ -181,6 +181,14 @@ You can change the order of received logs from the default descending order (new
 
 Each log row has an extendable area with its labels and parsed fields, for more robust interaction. For all labels we have added the ability to filter for (positive filter) and filter out (negative filter) selected labels. Each field or label also has a stats icon to display ad-hoc statistics in relation to all displayed logs.
 
+### Toggle parsed fields
+
+> **Note:** This feature is only available in Grafana 7.2+.
+
+If your logs are structured in `json` or `logfmt`, then you can show or hide parsed fields. Expand a log line and then click the eye icon to show or hide fields.
```

Comment:
Here and in the changelog you've used "in `json` or `logfmt`", I always thought it was "as `json` or `logfmt`". Would you say that both are correct or should I prefer one to the other for some reason?

Reply:
hmm I would say it's more of a semantics question but `as json` seems more correct to me. In the sense of json being a format not a container for something else but I suspect both are correct from English language perspective.